mui/PUBLISHING.md

```diff
@@ -0,0 +1,32 @@
+# Publishing Instructions
+
+## Automatic Publishing
+
+This package is published to NPM automatically by CircleCI when code is merged into the `dev` or `master` branches. To publish a new version, simply update the version in `package.json` and merge your code into the appropriate branch.
+- The `dev` branch will publish versions marked as `alpha` or `beta`.
+- The `master` branch will publish any version (`alpha`, `beta`, or `latest`).
+  In both cases, the code will only be published if the version number differs from the current version published under the respective dist tag.
+
+## Manually Publishing
+
+If you need to publish a package manually, you can run the following commands from the root folder:
+
+From the root: 
```

Comment:
This is redundant with the sentence above

Reply:
Removed the extra `for the root`